src/router/db.ts

```diff
@@ -106,7 +106,12 @@ db.post('/export', async (req, res) => {
 
 db.get('/list', async (req, res) => {
   try {
-    res.json(await iasql.list(dbMan.getUid(req.user), dbMan.getEmail(req.user), req.query.verbose === 'true'));
+    const uid = dbMan.getUid(req.user);
+    const email = dbMan.getEmail(req.user);
+    const dbs = await MetadataRepo.getDbs(uid, email);
+    res.json(dbs);
+    const recCount = dbs.map(d => d.recordCount).reduce((a, b) => a + b, 0);
+    telemetry.logDbList(uid, dbs.length, recCount);
```

Comment:
The dashboard polls every 30s the `/list` endpoint. Just commenting in case that could make some noise in the metrics